build.sh

```diff
@@ -1,12 +1,17 @@
 #!/usr/bin/env bash
 
-set -feuo pipefail
+set -o errexit -o pipefail -o noglob -o noclobber -o nounset
 IFS=$'\n\t'
 
 readonly IMAGES=(
   alpine-nix
+  alpine-nix-rails-nginx
+  alpine-nix-rails-nginx/example
 )
 
+rm -rf alpine-nix-rails-nginx/example
+(cd alpine-nix-rails-nginx && rails new example --template example-rails-template.rb --skip-bundle)
```

Comment:
Assuming a very pristine environment, `rails new` will not work as the gem is not installed, nor maybe ruby is available. I wounder if we want to run the `rails new` from *inside* a booted container instance to make sure all version requirements apply.